### PR TITLE
Add table with maintainers and backups of the code base parts

### PR DESCRIPTION
**Pull request by jkonecny12**
This table should have a few purpose:
- help community to find the correct person
- help us to cover all bigger parts of the project
- help us to not miss backups for all the code parts

We should fill all the missing parts and finalize what we expect from this table first.

### CODE REVIEW

File: CONTRIBUTING.rst [hunk 1]
```diff
@@ -257,3 +257,27 @@ Below is a list of pure community features, their community maintainers, and mai
...
+Network             rvykydal
+Storage             vponcova
+Payload             jkonecny
+Subscription        mkolman
+dbus                vponcova
+configuration files vponcova
+dasbus              vponcova
+simpleline          jkonecny
+initial-setup       mkolman
+=================== ========== ======
```
Comment (VladimirSlavik) [inline CONTRIBUTING.rst:283]: I think we're missing a lot of things here. Particularly pykickstart and Blivet. While these are technically not Anaconda, they are pretty much linked with everything we do. (I don't buy the "different red hat team" argument. That won't do any good to anyone reading this)

I'm not sure if kickstart tests should have an entry here, too?

File: CONTRIBUTING.rst [hunk 2]
```diff
@@ -257,3 +257,27 @@ Below is a list of pure community features, their community maintainers, and mai
...
+
+This section will provide you information about a contact person who is responsible for part of the 
+Anaconda code-base and or related projects. Anaconda team also use this list to avoid situation
+when does not have main maintainer or backup.
+
+Backup of the code should be familiar and develop parts similar way as maintainer does.
+Responsibility of a backup is to check all PRs created to this part of the Anaconda.
+
+=================== ========== ======
+Code part           maintainer backup
```
Comment (VladimirSlavik) [inline CONTRIBUTING.rst:272]: Github usernames might be a good thing to add.

**Review by VladimirSlavik — COMMENTED** [review]
Thanks for starting this!

File: CONTRIBUTING.rst [hunk 3]
```diff
@@ -257,3 +257,27 @@ Below is a list of pure community features, their community maintainers, and mai
...
+=================== ========== ======
+Network             rvykydal
+Storage             vponcova
+Payload             jkonecny
+Subscription        mkolman
+dbus                vponcova
+configuration files vponcova
+dasbus              vponcova
+simpleline          jkonecny
+initial-setup       mkolman
```
Comment (poncovka) [inline CONTRIBUTING.rst:282]: I would like to be a backup for the initial-setup.

File: CONTRIBUTING.rst (hunk 1 above)
Comment (VladimirSlavik) [reply CONTRIBUTING.rst:283]: Maybe also missing: networked storage, packaging & releases,...

Comment (jkonecny12) [reply CONTRIBUTING.rst]: Updated

File: CONTRIBUTING.rst (hunk 2 above)
Comment (jkonecny12) [reply CONTRIBUTING.rst]: Added

File: CONTRIBUTING.rst (hunk 3 above)
Comment (jkonecny12) [reply CONTRIBUTING.rst]: You are there

**Comment by jkonecny12** [comment]
**UPDATED:**
* implement comments above

**Comment by jkonecny12** [comment]
We will take another solution for this. Closing this PR.

**Pull request closed without merging**